components/camel-rest/pom.xml

```diff
@@ -38,6 +38,10 @@
             <groupId>org.apache.camel</groupId>
             <artifactId>camel-support</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-base</artifactId>
```

Comment:
Why do you have this dependency, we should never depend on this

Reply:
Removed as not more needed.

Was added because the endpoint registry as AbstractCamelContext overrides the signature as:

```java
EndpointRegistry<EndpointKey> getEndpointRegistry()
```

and `EndpointKey` is part of camel-base